src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -26,5 +27,147 @@ public partial class AppDomain
                 AppContext.UnhandledException -= value;  
             }  
         }  
+
+        public string DynamicDirectory => null;
+        public string FriendlyName => "DefaultDomain";
+        public int Id => DefaultADID;
+        public bool IsFullyTrusted => true;
+        public bool IsHomogenous => true;
+        public bool ShadowCopyFiles => false;
+        public event System.EventHandler DomainUnload
+        {
+            add  
+            {  
+                throw new PlatformNotSupportedException();
+            }  
+  
+            remove  
+            {  
+                throw new PlatformNotSupportedException();
+            }  
+        }
+
+        public event System.EventHandler<System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs> FirstChanceException
+        {  
+            add  
+            {  
+                AppDomainAugments.FirstChanceException += value;  
+            }  
+  
+            remove  
+            {  
+                AppDomainAugments.FirstChanceException -= value;  
+            }  
+        }  
+
+        public event System.EventHandler ProcessExit
+        {  
+            add  
+            {  
+                AppDomainAugments.ProcessExit += value;  
+            }  
+  
+            remove  
+            {  
+                AppDomainAugments.ProcessExit -= value;  
+            }  
+        }  
+
+        public void AppendPrivatePath(string path) { throw new PlatformNotSupportedException(); }
+        public string ApplyPolicy(string assemblyName) => AppDomainAugments.ApplyPolicy(assemblyName);
```

Comment:
This whole policy thing does not make sense on CoreCLR. It should just return what got passed into it - same as Mono: https://github.com/mono/mono/blob/b190f213a364a2793cc573e1bd9fae8be72296e4/mcs/class/corlib/System/AppDomain.cs#L1526